examples/asr/notebooks/6_VAD_using_NeMo.ipynb

```diff
@@ -0,0 +1,991 @@
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell.\n",
+    "!pip install wget\n",
+    "!pip install git+https://github.com/NVIDIA/apex.git\n",
+    "!pip install nemo-toolkit\n",
+    "!pip install nemo-asr\n",
+    "!pip install unidecode\n",
+    "\n",
+    "!mkdir configs\n",
+    "!wget -P configs/ https://raw.githubusercontent.com/NVIDIA/NeMo/master/examples/asr/configs/quartznet_vad_3x1.yaml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import some necessary libraries\n",
+    "import os\n",
+    "import argparse\n",
+    "\n",
+    "import copy\n",
+    "import math\n",
+    "import os\n",
+    "import glob\n",
+    "from functools import partial\n",
+    "from datetime import datetime\n",
+    "from ruamel.yaml import YAML"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "This VAD tutorial is based on the MatchboxNet model from the paper \"[MatchboxNet: 1D Time-Channel Separable Convolutional Neural Network Architecture for Speech Commands Recognition](https://arxiv.org/abs/2004.08531)\" with a modified decoder head to suit classification tasks.\n",
+    "\n",
+    "The notebook will follow the steps below:\n",
+    "\n",
+    " - Dataset preparation: Preparing Google Speech Commands dataset\n",
+    "\n",
+    " - Audio preprocessing (feature extraction): signal normalization, windowing, (log) spectrogram (or mel scale spectrogram, or MFCC)\n",
+    "\n",
+    " - Data augmentation using SpecAugment \"[SpecAugment: A Simple Data Augmentation Method for Automatic Speech Recognition](https://arxiv.org/abs/1904.08779)\" to increase number of data samples.\n",
+    " \n",
+    " - Develop a small Neural classification model which can be trained efficiently.\n",
+    " \n",
+    " - Model training on the Google Speech Commands dataset and Freesound dataset in NeMo.\n",
+    " \n",
+    " - Evaluation of error cases of the model by audibly hearing the samples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Preparation\n",
+    "\n",
+    "## Download the Freesound dataset\n",
+    "We will be using the freesound dataset (background categories) as our non-speech/background data. \n",
+    "We provide scripts and you can customize a lot by using it. Note that downloading this dataset may takes hours. \n",
+    "\n",
+    "1. Download provided downloading and resampling scripts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/download_resample_freesound.sh\n",
```

Comment:
Check if the file exists before using wget to download it again and again. wget will polute the workspace with copies of files. Also, create a temp src directory and push all these codes there.